src/EditorFeatures/Core.Wpf/Suggestions/PreviewChanges/PreviewChangesSuggestedAction.cs

```diff
@@ -31,21 +29,8 @@ private sealed partial class PreviewChangesSuggestedAction : SuggestedAction
             {
             }
 
-            public static async Task<SuggestedAction> CreateAsync(
-                SuggestedActionWithNestedFlavors suggestedAction, CancellationToken cancellationToken)
+            public static SuggestedAction Create(SuggestedActionWithNestedFlavors suggestedAction)
             {
-                var previewResult = await suggestedAction.GetPreviewResultAsync(cancellationToken).ConfigureAwait(true);
```

Comment:
@sharwell if you unconditionally return null here, the flyout still always shows up. Only the Preview Changes hyperlink never shows up.